ONSdigital/rm-collection-exercise-service
# Removing Sample V1 code

# What and why?
Now that sample v2 is enabled and sample is responsible for validation and distributing the sample file a lot of code can be deleted in collection exercise including 2 cron jobs and 4 database tables. This should help to simplify the service considerably as it is now only responsible for a collection exercise and all the events that happen on that.


# How to test?
Deploy to an environment, ensure the acceptance test passed and then create a new collection exercise and set ready for live

# Trello
https://trello.com/c/S5Jz26gc/441-s42-ce-sample-remove-feature-flag-code-from-collection-exercise
